pom.xml

```diff
@@ -1459,7 +1459,7 @@
       <dependency>
         <groupId>net.java.dev.jna</groupId>
         <artifactId>jna</artifactId>
-        <version>5.5.0</version>
+        <version>5.7.0</version>
```

Comment:
Seems the latest version is `5.14.0`. Should we try to upgrade to the latest one?

Reply:
Sure, let me update the PR.